README.adoc

```diff
@@ -940,13 +940,9 @@ In an OSGi context it will not work because JRuby needs some paths to find the A
 import static org.asciidoctor.Asciidoctor.Factory.create;
 import org.asciidoctor.Asciidoctor;
 
-JavaEmbedUtils.initialize(Arrays.asList("uri:classloader:/gems/asciidoctor-1.5.8/lib")); <1><2>
-
-Asciidoctor asciidoctor = create(this.getClass().getClassLoader()); <3>
+Asciidoctor asciidoctor = create(Arrays.asList("uri:classloader:/gems/asciidoctor-1.5.8/lib")); <1>
```

Comment:
Should this be `asciidoctor-{artifact-version}` instead of `asciidoctor-1.5.8`?

Reply:
Well, the gem within the final Jar is still named `asciidoctor-1.5.8`. I don’t know if it’s a typo but this is working in [SlideshowFX](https://github.com/twasyl/SlideshowFX/blob/master/slideshowfx-asciidoctor/src/main/java/com/twasyl/slideshowfx/markup/asciidoctor/AsciidoctorMarkup.java)

Reply:
Ah right :) Sorry for this.
Yes, here the different versioning shows up.
Maybe others might be confused too, like me and a comment highlighting this would be great.
Just saying that this has to use the version of the packaged version of Asciidoctor, not AsciidoctorJ.

Reply:
No problem at all :)
I just pushed a clarification within the README for the version number. Feel free to comment the modification :)